sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -365,8 +369,11 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
    * Negotiates the cbs claim for the ClientEntity.
    * @param setTokenRenewal - Set the token renewal timer. Default false.
    */
-  private async _negotiateClaim(setTokenRenewal?: boolean): Promise<void> {
-    this._logger.verbose(`${this._logPrefix} negotiateclaim() has been called`);
+  private async _negotiateClaim(
+    setTokenRenewal?: boolean,
+    operationOptions?: OperationOptionsBase
```

Comment:
Doesn't need to be optional (it's passed everywhere that I can see).

Reply:
Done.